src/mixins/raster-layer-point-mixin.js

```diff
@@ -448,7 +448,44 @@ export default function rasterLayerPointMixin(_layer) {
     return _layer
   }
 
+  // used for setting LastFilteredSize when point map first initialized
+  let _tot = null
+  _layer.tot = function(number) {
+    if (!arguments.length) {
+      return _tot
+    }
+    _tot = number
+    return _layer
+  }
+
+  _layer.getTotalRecordsAsync = function(cf) {
+    if (_layer.tot()) {
+      return Promise.resolve()
+    }
+    return cf
+        .dimension()
+        .groupAll()
+        .sizeAsync()
+        .then(tot => {
+          _layer.tot(tot)
+          return Promise.resolve()
+        })
+  }
+
   _layer._genVega = function(chart, layerName, group, query) {
+
+    // needed to set LastFilteredSize when point map first initialized
+    const cf = _layer.crossfilter()
+    if(!lastFilteredSize(cf.getId)){
+      if(cf.dimension){
+        _layer.getTotalRecordsAsync(cf).then(() => {
+          _layer.yDim().groupAll().valueAsync().then(value=>{
```

Comment:
just a little nit-picky, watch the spacing here: `value=>{`, IMO it's more readable to have `value => {` which is more consistent with the rest of our libraries as well.